// DO NOT EDIT
// This file was copied from
// https://raw.githubusercontent.com/tensorflow/tensorflow/master/tensorflow/core/example/feature.proto

// Use according to the original license.


syntax = "proto3";

package tensorflow_copy;

option cc_enable_arenas = true;
option java_outer_classname = "FeatureProtos";
option java_multiple_files = true;
option java_package = "org.tensorflow.example";
option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/example/example_protos_go_proto";
message BytesList {
  repeated bytes value = 1;
}
message FloatList {
  repeated float value = 1 [packed = true];
}
message Int64List {
  repeated int64 value = 1 [packed = true];
}
message Feature {
  oneof kind {
    BytesList bytes_list = 1;
    FloatList float_list = 2;
    Int64List int64_list = 3;
  }
}

message Features {
  map<string, Feature> feature = 1;
}
message FeatureList {
  repeated Feature feature = 1;
}

message FeatureLists {
  map<string, FeatureList> feature_list = 1;
}
